yt_dlp/extractor/n1.py

```diff
@@ -35,17 +35,47 @@ class N1InfoIIE(InfoExtractor):
     IE_NAME = 'N1Info:article'
     _VALID_URL = r'https?://(?:(?:\w+\.)?n1info\.\w+|nova\.rs)/(?:[^/?#]+/){1,2}(?P<id>[^/?#]+)'
     _TESTS = [{
-        # Youtube embedded
+        # YouTube embedded
         'url': 'https://rs.n1info.com/sport-klub/tenis/kako-je-djokovic-propustio-istorijsku-priliku-video/',
-        'md5': '01ddb6646d0fd9c4c7d990aa77fe1c5a',
+        'md5': '987ce6fd72acfecc453281e066b87973',
         'info_dict': {
             'id': 'L5Hd4hQVUpk',
             'ext': 'mp4',
             'upload_date': '20210913',
             'title': 'Ozmo i USO21, ep. 13: Novak Đoković – Danil Medvedev | Ključevi Poraza, Budućnost | SPORT KLUB TENIS',
             'description': 'md5:467f330af1effedd2e290f10dc31bb8e',
             'uploader': 'Sport Klub',
-            'uploader_id': 'sportklub',
+            'uploader_id': '@sportklub',
+            'uploader_url': 'https://www.youtube.com/@sportklub',
+            'channel': 'Sport Klub',
+            'channel_id': 'UChpzBje9Ro6CComXe3BgNaw',
+            'channel_url': 'https://www.youtube.com/channel/UChpzBje9Ro6CComXe3BgNaw',
+            'channel_is_verified': True,
+            'channel_follower_count': int,
+            'comment_count': int,
+            'view_count': int,
+            'like_count': int,
+            'age_limit': 0,
+            'duration': 1049,
+            'thumbnail': 'https://i.ytimg.com/vi/L5Hd4hQVUpk/maxresdefault.jpg',
+            'chapters': [{'start_time': 0.0, 'title': 'Uvod i pitanje za gledaoce', 'end_time': 57.0},
+                         {'start_time': 57.0, 'title': 'Za čime će Đoković najviše žaliti?', 'end_time': 162.0},
+                         {'start_time': 162.0, 'title': 'Pritisak', 'end_time': 366.0},
+                         {'start_time': 366.0, 'title': 'Tajming na udarcima, servis, više sati na terenu',
+                          'end_time': 510.0},
+                         {'start_time': 510.0, 'title': 'Propuštene brejk lopte, kako?', 'end_time': 670.0},
+                         {'start_time': 670.0, 'title': 'Publika nije bitna za ishod', 'end_time': 733.0},
+                         {'start_time': 733.0, 'title': 'Novakove suze na konferenciji i plan za budućnost',
+                          'end_time': 796.0},
+                         {'start_time': 796.0, 'title': 'Grend slem bitke u budućnosti – može li Novak?',
+                          'end_time': 972.0}, {'start_time': 972.0, 'title': 'Odgovor na pitanje', 'end_time': 1049}],
+            'categories': ['Sports'],
+            'tags': ['Novak Djokovic', 'Novak Đoković', 'US Open', 'US Open 2021', 'Daniil Medvedev', 'Danil Medvedev',
+                     'Sport Klub Tenis', 'Sport Klub', 'Tennis', 'Tenis'],
```

Comment:
To keep it a bit shorter
```suggestion
            'chapters': 'count:9',
            'categories': ['Sports'],
            'tags': 'count:10',
```